example/android/app/src/release/java/com/rnctvexample/ReactNativeFlipper.java

```diff
@@ -4,7 +4,7 @@
  * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
  * directory of this source tree.
  */
-package com.reactnativecollapsibletabviewdemos;
+package com.rnctvexample;
```

Comment:
now it's pulled from `./example/app.json` https://github.com/PedroBern/react-native-collapsible-tab-view/blob/b5f77dd4dc8681e23e431bf3b8e64c65a608cbc2/example/app.json#L15